freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/handler/SetPasswordHandler.java

```diff
@@ -31,14 +39,21 @@ public String selector() {
     @Override
     public void accept(Event<SetPasswordRequest> setPasswordRequestEvent) {
         SetPasswordRequest request = setPasswordRequestEvent.getData();
-        LOGGER.info("SetPasswordHandler accepting request {}", request);
+        Map<String, String> mdcContext = MDCBuilder.getMdcContextMap();
```

Comment:
Saving this context map is not relevant as `MDCCleanerThreadPoolExecutor` would cleanup it anyway after run, and on every accept the `FlowLogContextAspects` would set the context, but you can try it out

Reply:
also a log here would be nice as if stackService call fails, we won't have the accept log :)

Reply:
done